notebooks/experimental/semantic_operators.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Nit: "We" is generally not recommended by Google's tech writing style guide. Please rephrase as "The BigQuery DataFrames team implemented ..." or better yet "BigQuery DataFrames implements semantic operators as described in the "Lotus" paper"</p><p> </p><p>Nit: Remove "Without further ado, let's get started."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>